.gitignore

```diff
@@ -9,6 +9,7 @@ python/paddle/v2/fluid/tests/book/label_semantic_roles.inference.model/
 build/
 build_doc/
 *.user
+paddle/fluid/operators/distributed/send_recv.proto
```

Comment:
Should generate the file in the source directory, put it in the build directory. Can update this in another PR.